kakao/src/main/kotlin/com/agoda/kakao/delegates/ViewInteractionDelegate.kt

```diff
@@ -0,0 +1,25 @@
+package com.agoda.kakao.delegates
+
+import android.support.test.espresso.*
+import android.view.View
+import org.hamcrest.Matcher
+
+/**
+ * An interface duplicating the used in Kakao part of view interaction interface and providing original Espresso *Interactions.
+ * If you want to extend the base view interaction behavior, you should implement this interface, and specify it's
+ * factory in [com.kaspersky.klkakao.configurator.KakaoConfigurator]
```

Comment:
I think that we should remove any references cause CLA has been signed

Reply:
What is CLA?

Reply:
Contributor License Agreement